lib/src/showcase.dart

```diff
@@ -641,11 +775,54 @@ class _ShowcaseState extends State<Showcase> {
             descriptionTextDirection: widget.descriptionTextDirection,
             toolTipSlideEndDistance: widget.toolTipSlideEndDistance,
             toolTipMargin: widget.toolTipMargin,
+            tooltipActionConfig: _getTooltipActionConfig(),
+            tooltipActions: _getTooltipActions(),
           ),
         ],
       ],
     );
   }
+
+  List<Widget> _getTooltipActions() {
+    final actionData = (widget.tooltipActions?.isEmpty ?? true)
+        ? showCaseWidgetState.globalTooltipActions ?? []
+        : widget.tooltipActions ?? [];
+
+    final actionWidgets = <Widget>[];
+    for (final action in actionData) {
+      /// This checks that if current widget is being showcased and there is
+      /// no local action has been provided and global action are needed to hide
+      /// then it will hide that action for current widget
+      if (_showShowCase &&
+          action.hideActionForKeys.contains(widget.key) &&
+          (widget.tooltipActions?.isEmpty ?? true)) {
+        continue;
+      }
+      actionWidgets.add(
+        Padding(
+          padding: EdgeInsetsDirectional.only(
+            end: action != actionData.last
```

Comment:
I think you have to override `==` operator for `TooltipActionConfig`

Reply:
As discussed this logic is working properly so there is no need to override ```==``` method